### ISSUE TITLE
track_masks_data is all 0 of BestWishYsh/ConsisID-preview-Data

### ISSUE THREAD

**Issue by huiyang865** [PKU-YuanGroup/ConsisID#21]
Thanks to the great job, we find that the training data (BestWishYsh/ConsisID-preview-Data) you provided seemed a little problematic, for example,

All the track_masks_data in the directory (https://github.com/PKU-YuanGroup/ConsisID/tree/main/asserts/demo_train_data/dataname/track_masks_data/81FyMPk-WIS_segment_0_243_noFacePose_0_243/tracking_mask_results) are all 0 pixels.

And the tracking_mask_results of Zg-F_JweREs.7_segment_121_341-Scene-003_noFacePose_0_109 directory from BestWishYsh/ConsisID-preview-Data are also all 0 pixels.

Does the data provided  (BestWishYsh/ConsisID-preview-Data)  need to be re-run the preprocessed code (https://github.com/PKU-YuanGroup/ConsisID/tree/main/data_preprocess)?

**Comment by SHYuanBest** [comment]
Thanks for your interest about ConsisID. The `track_masks_data` is normal and not all 0 pixels. This is because it was saved in RGB format without binarization, which makes it appear black to the naked eye.
```
>>> from PIL import Image
>>> import numpy as np
>>> image_path = '/asserts/demo_train_data/dataname/track_masks_data/81FyMPk-WIS_segment_0_243_noFacePose_0_243/tracking_mask_results/1/annotated_frame_00000.png'
>>> image = Image.open(image_path)
>>> image_array = np.array(image)
>>> pixel_sum = np.sum(image_array)
>>> pixel_max = np.max(image_array)
>>> pixel_min = np.min(image_array)
>>> print(f"Pixel Sum: {pixel_sum}")
Pixel Sum: 5136
>>> print(f"Pixel Max: {pixel_max}")
Pixel Max: 1
>>> print(f"Pixel Min: {pixel_min}")
Pixel Min: 0
```

**Comment by bruce2233** [comment]
You should multiply 255

**Comment by SHYuanBest** [comment]
> You should multiply 255

Yes for visualization.

**Comment by LeoniusChen** [comment]
I have the same question. Does the data provided (BestWishYsh/ConsisID-preview-Data) need to be re-run the preprocessed code (https://github.com/PKU-YuanGroup/ConsisID/tree/main/data_preprocess)?

**Comment by LeoniusChen** [comment]
But in training, I met problems about "Using single face, but {idx} is multi person" and "No valid ID found: both valid_ids and backup_ids are empty". Maybe there is something incomplete in the training codes?

**Comment by SHYuanBest** [comment]
Yes, although the `data preprocess code` support multi person, but the current `training code` only support single face, and we are working hard to train a new model that support it, stay tuned.
